packages/webapp/public/locales/en/translation.json

```diff
@@ -2066,7 +2068,7 @@
   "BADGE": {
     "BETA": {
       "TITLE": "Beta",
-      "CONTENT": "Get a sneak peek at our new animal management feature—now in beta! Your feedback is crucial as we refine this feature. Please share any bugs or suggestions you encounter. We appreciate your help in improving liteFarm! You can learn more about beta <a>here</a>"
+      "ANIMALS_CONTENT": "Get a sneak peek at our new animal management feature—now in beta! Your feedback is crucial as we refine this feature. Please share any bugs or suggestions you encounter. We appreciate your help in improving liteFarm! You can learn more about beta <a>here</a>"
```

Comment:
Okay, it's a little too late but we'll want to avoid this in the future -- renaming strings that right now are only in CrowdIn -- because CrowdIn UX is not good for it.

I *think* the original translation was preserved (only and cryptically) in the AI suggestion, but its lost the usual authorship metadata and there appears to be no way to search for it 🫤

Reply:
Yeahhh, I knew it as I was doing it but figured we will need to be able to change key names. This one is particularly long I should have checked first. But was maybe hoping they hadn't got to it in the couple days badge was in for.